source/gui.cpp

```diff
@@ -337,8 +337,10 @@ void reshade::runtime::draw_ui()
 	imgui_io.Fonts->TexID = _imgui_font_atlas.impl.get();
 
 	// Scale mouse position in case render resolution does not match the window size
-	imgui_io.MousePos.x = _input->mouse_position_x() * (imgui_io.DisplaySize.x / _window_width);
-	imgui_io.MousePos.y = _input->mouse_position_y() * (imgui_io.DisplaySize.y / _window_height);
+	// imgui_io.MousePos.x = _input->mouse_position_x() * (imgui_io.DisplaySize.x / _window_width);
+	// imgui_io.MousePos.y = _input->mouse_position_y() * (imgui_io.DisplaySize.y / _window_height);
+	imgui_io.MousePos.x = static_cast<float>(_input->mouse_position_x());
+	imgui_io.MousePos.y = static_cast<float>(_input->mouse_position_y());
```

Comment:
Just one more thing: What's up with this?

Reply:
It does not work in the games i tested. Maybe it is dx9 related (did not check on dx10/dx11 games).
In my rig, the mouse does not do anything on the ui. Replacing the new code with the old one fix it.
For info, i tested on Mass Effect 2, Outlast, Half life 2...

Reply:
This is necessary to fix compatibility with downsampling. Check the values of `_window_width` and `_window_height` to make sure they contain correct values. If not will need to add some code that fixes those values, but not by reverting this.

Reply:
I'm guessing that this line here doesn't do the correct thing in those games: https://github.com/crosire/reshade/blob/81ee29a6456de172337c1af2064d92764e6db087/source/d3d9/runtime_d3d9.cpp#L242

Reply:
Also, I cannot reproduce it not working in Half-Life 2 ... Works fine for me there.

Reply:
Yes, in Half-Life 2 it works, but still not in Mass Effect 2, nor in Outlast, nor in Batman Arkham Asylum

Reply:
Works in Mass Effect 2 too. I don't have those other games to test.